CHANGELOG.md

```diff
@@ -29,6 +29,31 @@ After this, `Decidim::Proposals::ProposalEndorsement` and the corresponding coun
 
 ### Fixed
 
+- **decidim-forms**: Fixes a performance degradation when displaying forms in surveys. [\#5819](https://github.com/decidim/decidim/pull/5819)
```

Comment:
@microstudi watch out for these changes!

Reply:
what do you mean?

Reply:
Changelog had a lot of changes, unrelated to this PR. You already fixed, so I resolved the conversation 😄

Reply:
ah ah, it's a classic!